python/lsst/obs/pfs/isrTask.py

```diff
@@ -151,6 +163,15 @@ def validate(self):
         if self.doApplyGains:
             raise RuntimeError("doApplyGains must be False: gains are applied automatically.")
 
+        for detName, bboxStr in self.darkBBoxes.items():
+            try:
+                eval(bboxStr)           # can't set self.darkBBoxes[detName] here as it fails type validation
+                estr = ""
+            except Exception as e:
+                estr = str(e)           # it's clearer not to raise within the try block
```

Comment:
Not disagreeing, but why do you think it's clearer?

Reply:
Because it avoids the double traceback ("exception while processing exception" or something like that)

Reply:
I've never thought of that. Does `raise ValueError("...") from e` solve the problem?

Reply:
e doesn't exist outside the except block, and raising it in the block doesn't help.   But I clarified the comment